app/lib/constants/site_config.rb

```diff
@@ -215,6 +215,14 @@ module SiteConfig
         description: "Determines the mininum for the periodic email digest",
         placeholder: 2
       },
+      recaptcha_site_key: {
+        description: "Site key for Google reCAPTCHA, used for reporting abuse.",
+        placeholder: "..."
+      },
+      recaptcha_secret_key: {
+        description: "Secret key for Google ReCAPTCHA, used for reporting abuse.",
```

Comment:
_Non-blocking, nitpick_:
```suggestion
        description: "Secret key for Google reCAPTCHA, used for reporting abuse.",
```